src/pivot-table/components/helpers/get-item-size-handler.ts

```diff
@@ -31,24 +29,20 @@ export const getRowHeightHandler =
   };
 
 export const getColumnWidthHandler =
-  ({
-    list,
-    isLastRow,
-    layoutService,
-    allMeasuresWidth,
-    getMeasureInfoWidth,
-  }: ColumnWidthHandlerProps): ItemSizeHandler =>
+  ({ list, isLastRow, getRightGridColumnWidth }: ColumnWidthHandlerProps): ItemSizeHandler =>
   (colIndex: number) => {
     const cell = isLastRow ? list[colIndex] : Object.values(list)[colIndex];
-    const measureInfoCount = layoutService.layout.qHyperCube.qMeasureInfo.length;
+    // const measureInfoCount = layoutService.layout.qHyperCube.qMeasureInfo.length;
 
-    if (colIndex === 0 && cell?.x > 0) {
-      return ((cell.leafCount + cell.x) / measureInfoCount) * allMeasuresWidth;
-    }
+    // Dunno when this is true
```

Comment:
Can you add this:
> It's a bit of a special case but if you are on a different page then the first. Scroll down a bit and expand a node. The "first row" does not exist yet as data has only been fetched for the rows that are visible to the user.

As a comment here instead so that it does not get lost? I good to enough the specific scenario this logic is covering

Reply:
haha yep the current one is pretty bad